### PR TITLE
Change the lower bound of type annotations tag to 0x80

### PR DESCRIPTION
**Pull request by 1st1**
This way it's easy to test if a tag is an annotation or not
with a simple bitwise check: 0x80 & tag == 0x80.

### CODE REVIEW

File: edb/server/compiler/sertypes.py [hunk 1]
```diff
@@ -480,7 +481,9 @@ def parse(cls, typedesc: bytes) -> TypeDesc:
             desc = cls._parse(wrapped, codecs_list)
             if desc is not None:
                 codecs_list.append(desc)
-        return desc
+        if not codecs_list:
+            raise errors.InternalServerError('could not parse type descriptor')
+        return codecs_list[-1]
```
Comment (tailhook) [inline edb/server/compiler/sertypes.py:486]: Not sure how this is related to the issue...

**Review by tailhook — APPROVED** [review]

**Comment by tailhook** [comment]
Also, please fix protocol docs!

File: edb/server/compiler/sertypes.py (hunk 1 above)
Comment (1st1) [reply edb/server/compiler/sertypes.py:486]: It's in a separate commit

**Comment by 1st1** [comment]
> Also, please fix protocol docs!

I think I did, there's a change in the `docs/internals/protocol/typedesc.rst` file. Anything else?

**Comment by tailhook** [comment]
> > Also, please fix protocol docs!
> 
> I think I did, there's a change in the `docs/internals/protocol/typedesc.rst` file. Anything else?

No as long as I know. Was it originally there, and I've just missed that? Ah, nevermind.

**Pull request merged**